projects/storefrontapp-e2e-cypress/cypress/sample-data/checkout-with-variants-data.ts

```diff
@@ -0,0 +1,46 @@
+import { generateMail, randomString } from '../helpers/user';
+
+export const user = {
```

Comment:
We can just use the `user` in `checkout-flow`. No need duplicate this user, right?

Reply:
FIxed and committed new changes